static/styles/zulip.css

```diff
@@ -1440,7 +1440,7 @@ div.focused_table {
 }
 
 #message_edit_topic {
-    height: 20px;
+    height: 27px;
```

Comment:
Still looking at this, but I think this doesn't quite solve the problem. It makes the topic box bigger, rather than adding space under it. Below is what it looks like with this value set to 47px.

![sc1](https://cloud.githubusercontent.com/assets/890911/17580247/ff58c53a-5f51-11e6-93e1-497e10915e5f.png)